src/main/java/uk/gov/ons/ctp/response/action/service/ActionPlanJobService.java

```diff
@@ -88,20 +86,8 @@ private boolean hasActionableCases(ActionPlan actionPlan) {
     return actionCaseRepo.countByActionPlanFK(actionPlan.getActionPlanPK()) > 0;
   }
 
-  private boolean hasActionPlanBeenRunSinceLastSchedule(ActionPlan actionPlan) {
-    final Date lastExecutionTime =
-        new Date(nowUTC().getTime() - appConfig.getPlanExecution().getDelayMilliSeconds());
-    return !(actionPlan.getLastRunDateTime() == null
-        || actionPlan.getLastRunDateTime().before(lastExecutionTime));
-  }
-
-  /**
-   * Create a new ActionPlanJob and execute associated actions
-   *
-   * @param actionPlan Action plan to create and execute job for
-   */
-  private void createAndExecuteActionPlanJob(final ActionPlan actionPlan) {
-
+  @Transactional(propagation = REQUIRES_NEW)
```

Comment:
this transaction hint will be ignored when this method gets called internally (and it's more like a private method anyway)